lib/index.ts

```diff
@@ -146,3 +182,39 @@ async function getManifestFiles(
     })
     .filter((i) => i.contents !== "");
 }
+
+async function calcHashOfBinFiles(
+  docker: Docker,
+  files: string[],
+  options?: any,
```

Comment:
the options are given by the caller to this plugin, so I think it can be many things

Reply:
it definitely can - but I still expect ```options``` to be an object (maybe an empty object, but an object all the same). I don't expect it to ever be a string or an Error etc.
I also prefer it to be not undefined-able - that is, if no options are provided, we would just have an empty object, that's fine, no need for it to be undefined

Reply:
I dont understand something...
These options are coming from here:
![image](https://user-images.githubusercontent.com/16254901/78063336-f071f200-7387-11ea-9013-d4ea0b10a35b.png)

and it's defined as `options?: any`, so I'm not sure how I can change it

Reply:
it's defined as ```any``` because it was untyped to begin with.
a few weeks ago when our team resumed work on the docker-plugin I started adding types wherever I could, without going down rabbit holes too much. that is, where things were too difficult to type, I added ```any``` to mark them as "TODO" and moved on to things easier to type, incrementally improving things as I went.

the ```options``` object is one of the things I haven't gotten to covering yet, but basically we know it's an object that may or may not have things inside it.

so, what I mean is that if we're adding more code here, we shouldn't stick to the technical debt we have around typing in the plugin, but rather try to improve it, even just slightly every time.

makes sense?
if we can find a way to improve it, it would be great. if it's too much work right now then I guess we can disregard it... whatever makes sense to you

Reply:
as agreed, this change would be done later on as part of a bigger refactor to this codebase